### PR TITLE
Web 2.5 template

### PR DESCRIPTION
**Pull request by l-miskovsky**
Suggestions on how to simplify the whole standalone+batcher+template+frontend setup are welcome.

### CODE REVIEW

File: web-2.5/middleware/src/endpoints/write.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { ActionResult } from '../types';
+import { builder } from 'paima-sdk/paima-concise';
+import { getActiveAddress, postConciselyEncodedData } from 'paima-sdk/paima-mw-core';
+
+async function renamePlayer(name: string): Promise<ActionResult> {
+  const userWalletAddress = getActiveAddress();
+
+  const conciseBuilder = builder.initialize();
```
Comment (acedward) [inline web-2.5/middleware/src/endpoints/write.ts:8]: Now: `const conciseBuilder = builder.initialize();` should be 
  `const conciseBuilder = builder.initialize(undefined, ENV.CONCISE_GAME_NAME);` due to a change in the api.

File: web-2.5/post-batcher.mjs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  );
+
+  process.exit(1);
+}
+
+/**
+ * Example of submitting a gain experience input.
+ * This would be constructed on your server after verifying that user met the necessary conditions.
+ */
+const conciseBuilder = builder.initialize();
```
Comment (acedward) [inline web-2.5/post-batcher.mjs:62]: `const conciseBuilder = builder.initialize(undefined, process.ENV.CONCISE_GAME_NAME);`

File: web-2.5/frontend/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        </h1>
+      </header>
+      <main>
+        <p>
+          This example demonstrates Paima Engine capabilities when used in cooperation with
+          traditional centralized game servers.
+        </p>
+        <ul>
+          <li>
+            Admin functionality (experience gain) - can be modified and stored on chain only from
```
Comment (acedward) [inline web-2.5/frontend/src/App.tsx:69]: Perhaps add message to users know to search and run `post-batcher.mjs`

Comment (acedward) [reply web-2.5/frontend/src/App.tsx:69]: should it be something like `npm run post -- 0xwallet 100` ?

**Review by acedward — APPROVED** [review]

File: web-2.5/frontend/src/App.tsx (hunk 3 above)
Comment (l-miskovsky) [reply web-2.5/frontend/src/App.tsx:69]: Added :+1:

File: web-2.5/post-batcher.mjs (hunk 2 above)
Comment (l-miskovsky) [reply web-2.5/post-batcher.mjs:62]: It's currently failing if I do use some prefix. looking into it now

File: web-2.5/middleware/src/endpoints/write.ts (hunk 1 above)
Comment (l-miskovsky) [reply web-2.5/middleware/src/endpoints/write.ts]: added

File: web-2.5/post-batcher.mjs (hunk 2 above)
Comment (l-miskovsky) [reply web-2.5/post-batcher.mjs]: added

**Pull request merged**